### PR TITLE
adding Python 3.8 testing in the matrix

### PR DESCRIPTION
**Pull request by stonebig**
## Description
Adding Python 3.8 in the Matrix of test

## Status
- [ ] Ready to go

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -42,6 +43,12 @@ matrix:
       - PYTHON_VER=3.7
       - ENVIRON=FULL
 
+
+    - env:
+      - NAME="FULL 3.8"
+      - PYTHON_VER=3.8
+      - ENVIRON=FULL
```
Comment (dgasmith) [inline .travis.yml]: This should be `MINIMAL`.

File: .travis.yml [hunk 2]
```diff
@@ -47,7 +47,7 @@ matrix:
     - env:
       - NAME="FULL 3.8"
```
Comment (dgasmith) [inline .travis.yml]: We would need to change the name here too; however the FULL test worked so we can switch it back. Please do merge #122 and #124 together. Either PR is fine.

Comment (stonebig) [reply .travis.yml]: I don't know how to merge two PR with github web interface. please redo your own PR, if you want this. Glad to know Python-3.8 will work

Comment (dgasmith) [reply .travis.yml]: Closing the previous one and recommitting to this branch will work as well. I don't think you can merge through the GUI either.

**Pull request merged**